parquet-column/src/test/java/org/apache/parquet/parser/TestParquetParser.java

```diff
@@ -293,6 +298,36 @@ public void testIntAnnotations() {
     assertEquals(expected, reparsed);
   }
 
+  @Test
+  public void testIntegerAnnotations() {
+    String message = "message IntMessage {" +
+      "  required int32 i8 (INT(8,true));" +
+      "  required int32 i16 (INT(16,true));" +
+      "  required int32 i32 (INT(32,true));" +
+      "  required int64 i64 (INT(64,true));" +
+      "  required int32 u8 (INT(8,false));" +
+      "  required int32 u16 (INT(16,false));" +
+      "  required int32 u32 (INT(32,false));" +
+      "  required int64 u64 (INT(64,false));" +
+      "}\n";
+
+    MessageType parsed = MessageTypeParser.parseMessageType(message);
+    MessageType expected = Types.buildMessage()
+      .required(INT32).as(LogicalTypeAnnotation.intType(8, true)).named("i8")
```

Comment:
nit: I think, the static import of the method intType would help reading the code.